lib/pure/asyncnet.nim

```diff
@@ -69,13 +69,13 @@ type
   # TODO: I would prefer to just do:
   # AsyncSocket* {.borrow: `.`.} = distinct Socket. But that doesn't work.
   AsyncSocketDesc  = object
-    fd: SocketHandle
+    fd*: SocketHandle
```

Comment:
In case we ever want the implementation to change.

On Tuesday, 3 March 2015, Varriount <notifications@github.com> wrote:

> In lib/pure/asyncnet.nim
> <https://github.com/Araq/Nim/pull/2244#discussion_r25715811>:
>
> > @@ -69,13 +69,13 @@ type
> >    # TODO: I would prefer to just do:
> >    # AsyncSocket* {.borrow: `.`.} = distinct Socket. But that doesn't work.
> >    AsyncSocketDesc  = object
> > -    fd: SocketHandle
> > +    fd*: SocketHandle
>
> Why does this need a getter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Araq/Nim/pull/2244/files#r25715811>.
>